playbooks/ovs_2_ml2_cleanup_remnants.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+# a cleanup script to run after a host has been migrated from OVS to ml2 / linux bridge
+
+- hosts: all
+  serial: 10
```

Comment:
Kinda yeah. Serial says run 20 hosts through the first task, then through the second task and then through the third task, etc... until no more tasks. Then run the next 20 through each task. The amount of parallelism is governed by the forks option. If serial is higher than forks, you get X forks running in parallel. If serial is lower than forks, then you get limited to the serial amount.

 I'm just curious why there is a need to make any distinction here in this play.